fs/expose/dokan/__init__.py

```diff
@@ -262,12 +309,13 @@ def wrapper(self,*args):
 
 MIN_FH = 100
 
+
 class FSOperations(object):
     """Object delegating all DOKAN_OPERATIONS pointers to an FS object."""
 
-    def __init__(self, fs, fsname="Dokan FS", volname="Dokan Volume"):
+    def __init__(self, fs, fsname="NTFS", volname="Dokan Volume"):
```

Comment:
@lurch It is more safe to put NTFS 😃 There is no doc about the difference on this "features" for FAT vs NTFS.
This come from a issue we had that application from the dokan device could not create a socket on a FS different that NTFS